### DISCUSSION TITLE
Does firely .net SDK support patient data from multiple organizations? like EPIC Care Everywhere

### DISCUSSION THREAD

**Discussion by fayazshaik07** [FirelyTeam/firely-net-sdk#2703] [Q&A]
We're actively using Firely .Net SDK for EPIC FHIR Integration in our Application. I have a scenario where I need to get patient's past medical history if it is spread across multiple organizations?
Does Firely .Net sdk team know how to achieve it? If possible , could you please suggest.
@mbaltus  your expertise is needed. Thanks in advance.

**Comment by mmsmits** [comment]
This is not an issue, but something for the discussion topic.

**Reply by mmsmits** [reply]
This is not a question related to the SDK I think. 
If you need help with FHIR implementations, I would suggest to subscribe to http://chat.fhir.org.
If you want to know how to use a FhirClient using the SDK to send requests to a FHIR Server, please read our documentation. 
https://docs.fire.ly/projects/Firely-NET-SDK/en/latest/client/introduction.html
